frontend/src/components/bs-space/bs-space.tsx

```diff
@@ -0,0 +1,90 @@
+import { IconName } from 'common/enums/enums';
+import { IconButton } from 'components/common/common';
+import { useAppDispatch, useEffect, useParams } from 'hooks/hooks';
+import { ObjectsTable } from './components/components';
+import React, { FC } from 'react';
+import styles from './styles.module.scss';
+import { BSSpace as BSSpaceActions } from 'store/actions';
+
+const BSSpace: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(
+      BSSpaceActions.loadObjects({
+        filter: {
+          from: 0,
+          count: 5,
+        },
+        id: id as string,
+      }),
+    );
+  }, [dispatch]);
+
+  const handleObjectsReload = (): void => {
+    dispatch(
+      BSSpaceActions.loadObjects({
+        filter: {
+          from: 0,
+          count: 5,
+        },
+        id: id as string,
+      }),
+    );
+  };
+
+  const handleObjectDownload = (objectId: string): void => {
+    dispatch(
+      BSSpaceActions.downloadObject({
+        spaceId: id as string,
+        objectId,
+      }),
+    );
+  };
+
+  const handleObjectUpload = (e: React.FormEvent<HTMLInputElement>): void => {
+    const [file] = e.currentTarget.files as FileList;
+    const hasFiles = Boolean(file);
+
+    if (!hasFiles) {
+      return;
+    }
+
+    const formData = new FormData();
+    formData.append('file', file as File);
```

Comment:
```suggestion
    formData.append(FormDataCommonKey.FILE, file as File);
```
create an enum FormDataCommonKey and out it for both frontend and back part instead of a normal string